barcode/example/main.dart

```diff
@@ -82,7 +82,7 @@ void main() {
 
   buildBarcode(
     Barcode.ean2(),
-    '05',
+    '12',
```

Comment:
A unit test would be better than changing this

Reply:
I have removed this part of the edit in [b286841](https://github.com/DavBfr/dart_barcode/pull/65/commits/b2868410d5f09fdc5ec53684aa63439cdd4d29b7).
However, the problem with this fix is the barcode drawing and has nothing to do with data conversion. I have not learned to write unit tests for graphical interfaces.

Reply:
Sorry, I have known how to test barcode graphics from other unit test. 
I will add this unit test next week.
 Thanks.